packages/components/src/test/data/users.ts

```diff
@@ -186,9 +189,44 @@ export const TEST_USER_APP_ADMIN = new User({
         PermissionTypes.Insert,
         PermissionTypes.Update,
         PermissionTypes.ManagePicklists,
+        PermissionTypes.ManageSampleWorkflows,
         PermissionTypes.Admin,
         PermissionTypes.UserManagement,
         PermissionTypes.ApplicationAdmin,
         PermissionTypes.AddUser,
     ]),
 });
+
+export const TEST_USER_STORAGE_DESIGNER = new User({
+    canDelete: false,
+    canDeleteOwn: false,
+    canInsert: false,
+    canUpdate: false,
+    canUpdateOwn: false,
+    isAdmin: false,
+    isAnalyst: false,
+    isDeveloper: false,
+    isGuest: true,
+    isRootAdmin: false,
+    isSignedIn: false,
+    isSystemAdmin: false,
+    isTrusted: false,
+    permissionsList: List<string>([PermissionTypes.Read, PermissionTypes.DesignStorage]),
```

Comment:
I still find it odd that we don't just give Storage roles read permission...

Reply:
One issue with doing that is we lose granularity since 'Read' grants permission to read everything in the container and we've seen that that is not always wanted. We'll likely create a separate role for Sample Stewards that grants the permissions given to editors and/or designers, but one could also imagine that there's a "Sample Editor" role (or something like that) and a sample steward is assigned Storage Editor and Sample Editor to get the functionality they need.